app/src/components/register/DropZone.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useCallback } from 'react';
+import * as styles from './DropZone.module.scss';
+import { useDropzone } from 'react-dropzone';
+import { FilesContext } from './RegisterForm';
+import { saveToIPFS } from '../../helper/ipfs';
+
+const DropZone: React.FC<{popup?: true; callback: (hash: string) => void}> = ({ popup, callback }) => {
+  const { files } = React.useContext(FilesContext);
+
+  const onDrop = useCallback((acceptedFiles: Blob[]) => {
+    const reader = new FileReader();
+    reader.addEventListener('loadend', function () {
+      saveToIPFS([reader.result], callback);
+    });
+    acceptedFiles.forEach((blob) => reader.readAsArrayBuffer(blob));
+  }, [callback]);
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({ onDrop });
+
+  const isFileOver = isDragActive || files.image !== '';
+  const isComplete = files.image !== '';
+
+  return (
+    <>
+      {popup
+        ? <div className={styles.popup}>
+          <span className={styles.message}>
+      Artifact Image<br />
+            <small>Drop your image into the space below</small>
```

Comment:
Assuming that this is being reused at least for documents and images upon artifact registration surely this text needs changing? Also: this component is likely to need to be accessed from within our expanded documents view, not just register related stuff: should it move to a slightly more general folder?

Reply:
Doesn't seem like this change was made?

Reply:
Maybe I'm very confused: what uses this dropzone?

Reply:
Okay, turns out the question I mean is why do we use this class for the image and not for the file? Should the image stuff not be built like the file drop thing is or vice versa rather than the two being constructed differently?

Reply:
Dropzone is only used for the image.
We could technically use dropzone for the Register Button, but it is just a button, so I don't see what the gain is

Reply:
No I mean the documents button that is also a droppable location?

Reply:
Ah wait, I see my mistake: documents isn't I just think it should be. Does dropzone only allow a single thing to be attached? If so I'm happy to just approve

Reply:
Currently, You could expand it to be multiple relatively easily